answerking_app/tests/test_unit/test_unit_category_serializer.py

```diff
@@ -75,8 +75,8 @@ def test_cat_serializer_products_field_content(self):
             name=self.test_cat_data["name"]
         )
         test_serializer_data: ReturnDict = CategorySerializer(test_cat).data
-        expected: dict = dict(*test_cat.products.values("id"))
-        actual: dict = dict(*test_serializer_data["products"])
+        expected = dict(*test_cat.products.values("id"))["id"]
```

Comment:
why get rid of the types?

Reply:
This test broke and had to make changes just forgot the types. They are `int` 's now and added back in